README.md

```diff
@@ -19,6 +19,8 @@ monitor topic. This allows Kafka Monitor to detect performance issue on every
 broker without requiring users to manually manage the partition assignment of
 the monitor topic.
 
+Kafka Monitor is used in conjunction with different middle-layer services such as linkedin-kafka-clients and Kafka REST proxy in order to monitor single clusters, pipeline desination clusters, as well as Kafka REST clusters, as done at Linkedin engineering for real-time cluster healthchecks.
```

Comment:
Our rest proxy is not available to the public nor is linkedin-kafka-clients.  Why mention this at all?

Reply:
linkedin-kafka-clients should be li-apache-kafka-clients.
https://github.com/linkedin/li-apache-kafka-clients